src/cycle-times.coffee

```diff
@@ -57,10 +57,10 @@ module.exports = (robot) ->
     msg.send "Timezone offset is set to #{tzOffset}. I hope you know what you are doing."
 
   robot.respond /cycle set offsetname (.*)/i, (msg) ->
-    tzOffset = msg.match[1]
+    tzName = msg.match[1]
     msg.send "Timezone offset name is set to #{tzName}. I hope you know what you are doing."
 
-  robot.respond /(septi)?cycle\s*([0-9])?/i, (msg) ->
+  robot.respond /(septi)?cycle\s*([0-9])?$/i, (msg) ->
```

Comment:
Looks like this change is causing the tests to fail. Please update the test to match.